File: angular/src/app/models/intake-form.model.ts
import { IntakeFormType } from './enums/intake-form-type.enum';
import { IntakeStatus } from './enums/intake-status.enum';
import { ICD10Code } from './icd10-code.model';
import { Question } from './question.model';

export class IntakeForm {
    public intakeFormId: string;
    public patientId: string;
    public physicianId: string;
    public documentId: string;
    public status: IntakeStatus;
    public intakeFormType: IntakeFormType;
    public questions: Question[];
    public ICD10Codes: ICD10Code[];
    public HCPCSCode: string;
    public duration: string;
    public physicianNotes: string;
    public deniedReason: string;
    public product: string;
    public vendorBilled: boolean;
    public vendorPaid: boolean;
    public physicianPaid: boolean;
    public createdOn: string;
    public modifiedOn: string;
}

File: dotnet/PhysiciansReach/PR.Models/IntakeFormModel.cs
using PR.Constants.Enums;
using System;
using System.Collections.Generic;

namespace PR.Models
{
    public class IntakeFormModel
    {
        public int IntakeFormId { get; set; }

        public int PatientId { get; set; }

        public int? PhysicianId { get; set; }

        public IntakeFormType IntakeFormType { get; set; }

        public IntakeFormStatus Status { get; set; }

        public List<ICD10CodeModel> ICD10Codes { get; set; }

        public List<HCPCSCodeModel> HCPCSCodes { get; set; }

        public string Duration { get; set; }

        public string PhysicianNotes { get; set; }

        public DateTime CreatedOn { get; set; }

        public DateTime ModifiedOn { get; set; }

        public List<QuestionModel> Questions { get; set; }

        public string AdditionalDrNotes { get; set; }

    }
}
